owncloudApp/src/main/java/com/owncloud/android/presentation/sharing/shares/PublicShareDialogFragment.kt

```diff
@@ -310,6 +320,18 @@ class PublicShareDialogFragment : DialogFragment() {
         }
     }
 
+    private fun initPasswordChangeInputListener() {
+        binding.shareViaLinkPasswordValue.doOnTextChanged { text, _, _, _ ->
+            capabilities?.passwordPolicy?.let { passwordPolicy ->
+                requirementsPasswordPolicy(text.toString(), passwordPolicy)
+            } ?: binding.saveButton.also {
+                if(binding.shareViaLinkPasswordSwitch.isChecked){
```

Comment:
Again, be careful with code formatting... Two spaces are left here and KtLint won't approve this
```suggestion
                if (binding.shareViaLinkPasswordSwitch.isChecked) {
```

Reply:
Done